libcnb/src/tracing.rs

```diff
@@ -0,0 +1,136 @@
+use libcnb_data::buildpack::Buildpack;
+use opentelemetry::{
+    global,
+    trace::{Span as SpanTrait, Status, Tracer, TracerProvider as TracerProviderTrait},
+    KeyValue,
+};
+use opentelemetry_sdk::{
+    trace::{Config, Span, TracerProvider},
+    Resource,
+};
+use std::path::Path;
+
+const TELEMETRY_EXPORT_ROOT: &str = "/tmp/cnb-telemetry";
```

Comment:
This hardcoded path still makes me a bit uneasy. But I also don't have a great solution either. Could we:

- Add `#[cfg(target_family = "unix")]` to compilation will at least fail on non-unixes. This way, it wouldn't compile for Windows instead of failing with an unsupported path at runtime.
- Include `libcnb` in the path since it is highly libcnb specific.

Reply:
A code comment here explaining what the seemingly arbitrary path is all about would be useful too :-)

Reply:
Addressed in [7bbf0e7](https://github.com/heroku/libcnb.rs/pull/723/commits/7bbf0e710aaa80b4925453509fe4520a6a7523c1).